app/post/postDetailsDirective.js

```diff
@@ -279,6 +280,12 @@
             return post;
         };
 
+        postDetailsCtrl.showImage = function() {
+            var imageEmpty = postDetailsCtrl.post.photo_url === "";
+            var imageNull = postDetailsCtrl.post.photo_url === null;
+            return !(imageEmpty || imageNull);
```

Comment:
quando vc faz um not em uma expressão com múltiplas variáveis é melhor aplicar o not logo, no caso !imageEmpty && !imageNull